### ISSUE TITLE
Upload of files to my work appears to be successful, but no files are associated or displayed

### ISSUE THREAD

**Issue by hannahfrost** [projecthydra-labs/hyku#430]
When I drag and drop files, they are processed as if uploaded. After saving, the page indicated that there are no files associated with the work.

**Comment by hannahfrost** [comment]
@mjgiarlo OK if I put this in the Ready column to be worked on soon?

**Comment by mjgiarlo** [comment]
@hannahfrost yep!

**Comment by jcoyne** [comment]
I'm seeing this message in the logs:
```
I, [2016-11-07T16:07:14.535112 #24403]  INFO -- : [ActiveJob] [AttachFilesToWorkJob] [4c76f921-ae7f-48a3-ab7b-7feb4d46851f] Performing AttachFilesToWorkJob from Async(ingest) with arguments: #<GlobalID:0x00564305897468 @uri=#<URI::GID gid://lerna/GenericWork/3484zg88m>>, [#<GlobalID:0x005643058966a8 @uri=#<URI::GID gid://lerna/Sufia::UploadedFile/8>>]
```
which is logged here:
https://github.com/rails/rails/blob/92703a9ea5d8b96f30e0b706b801c9185ef14f0e/activejob/lib/active_job/logging.rb#L72

I think this indicates that we are using the async driver rather than SQS.

**Comment by jcoyne** [comment]
I think we need to configure `Settings.active_job.queue_adapter` on AWS.  @cbeer can you show me how that's done?

**Comment by cbeer** [comment]
e.g. https://github.com/hybox/aws/blob/master/templates/webapp.json#L428-L431 + https://github.com/hybox/aws/blob/master/templates/worker.json#L253-L256.

**Comment by jcoyne** [comment]
Uploads were working as of Oct 18th, but seemed to stop working before Oct 27th.

**Comment by jcoyne** [comment]
Fixed by #473
